tests/manage/z_cluster/test_must_gather.py

```diff
@@ -45,3 +48,27 @@ def test_must_gather(self, mustgather, log_type):
         """
         mustgather.log_type = log_type
         mustgather.validate_must_gather()
+
+    @tier4
+    @tier4a
+    @bugzilla("1770199")
+    @pytest.mark.polarion_id("OCS-2328")
+    def test_must_gather_worker_node_down(self, mustgather, nodes):
+        """
+        Collect must-gather OCS logs when a worker node is down
+
+        """
+        logger.info("Get all worker nodes and choose random from worker nodes list")
+        worker_nodes = get_worker_nodes()
+        worker_node = worker_nodes[randint(0, len(worker_nodes) - 1)]
+
+        logger.info(f"Stop {worker_node} worker node")
+        nodes.stop_nodes(get_node_objs([worker_node]))
+
+        logger.info("Collect must gather and check content")
+        mustgather.collect_must_gather()
+        mustgather.log_type = "CEPH"
+        mustgather.validate_must_gather()
+
+        logger.info(f"Start {worker_node} worker node")
+        nodes.start_nodes(get_node_objs([worker_node]))
```

Comment:
after bringing back the node I think it will be good to add a check to verify that all OCS pods reached running state,